### ISSUE TITLE
fix occil/sqlite3

### ISSUE THREAD

**Issue by LADSoft** [LADSoft/OrangeC#955]
sqlite3 was recently updated to the latest package, however, there are a couple of issues with occil.    First, sizeof() is used to initialize a data member, which doesn't result in a constant value with occil.    Second, when compiling 'shell.exe' with occil, a stack underflow is generated by the pelib stack.

the offending sizeof line was changed to allow occil to work, and the build of shell.exe is being ignored at present

**Comment by GitMensch** [comment]
>`sizeof()` doesn't result in a constant value with occil

sounds pretty bad, but not that hard to fix

**Comment by LADSoft** [comment]
well the deal is, to handle sizeof there is some sort of sizeof() operator in CIL.   I had to start using it for compatibility with various disassemblers a few years ago...   and sizeof() used in code blocks does work as expected,  but I didn't properly handle the case where sizeof() was used in a data initializer.   It needs to be compacted down to a constant again at that point...

But you are right it shouldn't be *too* hard to fix it lol..   I just need to get on with the release process right now because i want to get #603 out of the way, so I punted.    I'm halfway expecting this to rear its ugly head again when I start trying to compile libcob with occil though :smile:
